website/redirects.js

```diff
@@ -100,7 +100,17 @@ module.exports = [
   },
   {
     source: '/help/admin-ui/sessions',
-    destination: '/docs/common-workflows/manage-sessions',
+    destination: '/docs/concepts/domain-model/sessions',
+    permanent: false,
+  },
+  {
+    source: '/help/admin-ui/credential-stores',
+    destination: '/docs/concepts/domain-model/credential-stores',
+    permanent: false,
+  },
+  {
+    source: '/help/admin-ui/credential-sources',
```

Comment:
Is this an early pivot from credential-libraries to credential-sources?

Reply:
Yeah.  It's just the redirect URL which users will almost never see.

Reply:
Gotcha. Just checking.